css/src/tokens/colors.scss

```diff
@@ -224,4 +224,26 @@ $color-index-background-glow-high-contrast: 8;
 // 	$box-shadow: nth($color-set, $color-index-box-shadow);
 //}
 
+$gradients: (
+	'vivid': (
+		$gradient-vivid-start,
+		$gradient-vivid-end
+	),
+	'purple-blue': (
+		$gradient-text-purple,
+		$gradient-text-blue
+	)
+);
+
+$gradient-color-start-index: 1;
+$gradient-color-end-index: 2;
+
+// @each $name, $gradient in $gradients {
+// 	$colorStart: nth($gradient, $gradient-color-start-index);
+// 	$colorEnd: nth($gradient, $gradient-color-end-index);
+
+// 	.gradient-text-#{$name} {
+// 		@include gradient-text($colorStart, $colorEnd);
+// }
```

Comment:
Should this comment be removed or uncommented?

Reply:
@BenDMyers  I was matching the color loop comment above it describing how to use it within the context of that file. but I can remove it also